Airports/EGCB/Basic.txt

```diff
@@ -1,3 +1,3 @@
 Manchester Barton
 N053.28.17.000 W002.23.21.000
-122.700
```

Comment:
@itsbenny99 do you use GitHub desktop? Can you open this branch and try and revert the changes to this file? Otherwise it will show as last updated by this PR. Cheers